schema/spacewalk/upgrade/susemanager-schema-4.0.11-to-susemanager-schema-4.0.12/051-suseContentEnvironmentTarget.sql.oracle

```diff
@@ -0,0 +1 @@
+-- intentionally blank, corresponds to 20-environment-target-state.sql.postgresql
```

Comment:
Maybe this was a copy/paste leftover and it was intended to be `051-suseContentEnvironmentTarget.sql.postgresql`?